docs/aerodromes/classc/EastSale.md

```diff
@@ -4,7 +4,149 @@
+| Runway 22  | E2 or Eastern Check Bays         | J                                |
+| Runway 04  | J                                | D2                               |
+
+## Helicopter Operations
+- **Helicopter Circuits**:  
+  - When RWY09/27 is being used for fixed-wing circuit training, it is preferred that helicopters utilise threshold runway 04 (runway 09/27 direction) for landing and take-off to increase segregation between final approach and upwind segments.
+  - When helicopters require use of TWY A (Pad Alpha) for circuit training, base turns should be sequenced to avoid conflict during the final approach segment.
+
+## Lanes
+- **Lanes Overview**:  
+  The 16 individual training areas within the ESL military airspace are separated by 4 outbound lanes.
+  These lanes are used aircraft transiting to/from exterior training areas or for entry and exit of ESL military airspace.
+  Lanes are defined by GNSS waypoints situated at 12, 35, and 50 NM from YMES AD.
+  
+- **Outbound Lanes**:
+
+| Lane Direction | 12NM GNSS Waypoint | 35NM GNSS Waypoint | 50NM GNSS Waypoint | Bearing |
+| -------------- | ------------------ | ------------------ | ------------------ | ------- |
+| Northern       | LEKEM              | GONEB              | LANOS              | 359     |
+| Eastern        | VEMDA              | KADRU              | TAVET              | 090     |
+| Southern       | SABAX              | LUTUK              | NOLOX              | 180     |
+| Western        | DUGAD              | LERKO              | DUNNE              | 270     |
+
+- **Vertical Dimensions**:  
+  Lanes extend from SFC or the base of restricted airspace to `F160`.
+
+- **Departure Instructions**:  
+  For departure instructions with a heading assignment, ATC must ensure that an assigned heading or a climb requirement is provided to segregate or separate with STA activity.
+
+## Level Assignment
+- `F160` or `RFL` whichever is lower for fixed-wing aircraft.
+- `A040` or `RFL` whichever is lower for rotary-wing aircraft.
+
+## Coordination
+
+### Auto Release
+Auto release is not utilised at East Sale. 'Next' coordination is required from ES ADC to ES TCU for all aircraft.
+
+!!! example
+    <span class="hotline">**ES ADC** -> **ES TCU**</span>: "Next, ASY01, runway 09"  
+    <span class="hotline">**ES TCU** -> **ES ADC**</span>: "ASY01, Assigned Heading Left 030, unrestricted"  
+    <span class="hotline">**ES ADC** -> **ES TCU**</span>: "Assigned Heading Heading Left 030, ASY01" 
+
+Helicopters departing from helicopter spots will be treated as if departing from the duty runway.
+
+!!! example
+    *PSDN14 is a VFR helicopter departing from the threshold of RWY04 (in the direction of runway 27)*  
+    <span class="hotline">**ES ADC** -> **ES TCU**</span>: "Next, PSDN14, runway 27"  
+    <span class="hotline">**ES TCU** -> **ES ADC**</span>: "PSDN14, right turn, unrestricted"  
+    <span class="hotline">**ES ADC** -> **ES TCU**</span>: "Right turn, PSDN14" 
```

Comment:
Are we able to define these helicopter spots? Or is it just Pad Alpha as mentioned above?

Reply:
From what I've seen in their manual just the RWY04 and pad alpha. I don't think they have a huge helicopter area like nowra.